scripts/new-component/create-component-files.js

```diff
@@ -0,0 +1,42 @@
+const fs = require('fs');
+const path = require('path');
+const yargs = require('yargs');
+const nunjucks = require('nunjucks');
+
+const componentName = yargs.argv._[0];
+const themes = [ 'default', 'bootstrap', 'material', 'classic' ];
+const files = fs.readdirSync(path.resolve(path.dirname(process.argv[1]) + '/templates'));
+
+nunjucks.configure(path.resolve(__dirname, './', 'templates'), { autoescape: false });
+
+if (!componentName) {
+    throw 'Pass the component name as a parameter!';
+}
+
+themes.forEach((theme) => {
+    const dist = './packages/' + theme + '/scss/';
+    const data = {
+        componentName: componentName,
+        theme: theme
+    };
+
+    ensureDirSync(path.join(dist + componentName));
```

Comment:
`fs.mkdirSync(... , { recursive: true });` is sufficient.  If it fails, it fails.